content/en/docs/concepts/configuration/manage-resources-containers.md

```diff
@@ -237,6 +238,73 @@ are available in your cluster, then Pod resource usage can be retrieved either
 from the [Metrics API](/docs/tasks/debug/debug-cluster/resource-metrics-pipeline/#metrics-api)
 directly or from your monitoring tools.
 
+### Considerations for memory backed `emptyDir` volumes {#memory-backed-emptydir}
+
+{{< caution >}}
+If you do not specify a `sizeLimit` for `emptyDir` nor a memory limits
+(`spec.containers[].resources.limits.memory`) for pod, you can create a memory
+backed `emptyDir` with node allocatable size unintentionally. Also, user can
+create multiple memory backed `emptyDir`s on the same node, each set to a node
+allocatable size.  And what Kubernetes sees as memory usage is the amount that
+is actually consuming the memory　backed `emptyDir`, so even after you create a
+memory backed `emptyDir` with node allocatable size, the other Pods can be
+scheduled to the same node.  As a result, multiple memory backed `emptyDir`s
+with node allocatable size can be created on the same node.
+{{< /caution >}}
+
+From the perspective of memory management, there are some similarities between
+when a process uses memory as a work area and when using memory-backed
+`emptyDir`. But when using memory as a volume like memory-backed `emptyDir`,
+there are additional points below that you should be careful.
+
+* Files stored on a memory-backed volume are almost entirely managed by the
+  user application. Unlike when used as a work area for a process, you can not
+  rely on like garbage collection by the runtime for language.
+* The purpose of writing files to a volume is to save data or pass it between
+  applications. Deletion of files may not be considered or may depend on other
+  applications, then tend to persist for long time as files accumulate or are
+  not deleted as intended.
+* The memory-backed `emptyDir` is useful when writing large amounts of logs
+  continuously or passing large image files because of its performance.
+  However, memory is very small compared to disks, and using a large amount of
+  memory will affect the normal operation of the node, so you must be very
+  careful to use the minimum amount necessary.
+
+If you use memory backed `emptyDir` without appropriate settings, e.g. without
```

Comment:
minor: "If you use a memory backed `emptyDir` without appropriate limits, one or more of the following may occur:"